splice_machine/src/main/java/com/splicemachine/derby/stream/control/ControlDataSet.java

```diff
@@ -114,6 +107,23 @@ public int partitions() {
         }
     }
 
+    @Override
+    public DataSet<V> orderBy(OperationContext operationContext, int[] keyColumns, boolean[] descColumns, boolean[] nullsOrderedLow) {
+        //operationContext.pushScopeForOp(OperationContext.Scope.SORT_KEYER);
+        KeyerFunction f=new KeyerFunction(operationContext,keyColumns);
+        PairDataSet pair=keyBy(f);
+        //operationContext.popScope();
+
+        //operationContext.pushScopeForOp(OperationContext.Scope.SHUFFLE);
+        PairDataSet sortedByKey=pair.sortByKey(new RowComparator(descColumns,nullsOrderedLow),
+                OperationContext.Scope.SORT.displayName(), operationContext);
+        //operationContext.popScope();
+
+        //operationContext.pushScopeForOp(OperationContext.Scope.READ_SORTED);
+        return sortedByKey.values(OperationContext.Scope.READ_SORTED.displayName());
+        //operationContext.popScope();
+    }
```

Comment:
Remove code that were commented out.

Reply:
I left that in deliberately.  It is not clear to me why that is commented out, etc.